plotly/tools.py

```diff
@@ -2332,7 +2332,7 @@ def create_candlestick(open, high, low, close,
 
     @staticmethod
     def create_distplot(hist_data, group_labels,
-                        bin_size=1., curve_type='kde',
+                        bin_size=[1.], curve_type='kde',
```

Comment:
This default argument is mutable, which is subject to 🐛s http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments

I'd suggest making it `(1.,)` or `None` and checking if it's `None` in the code.